### ISSUE TITLE
Typed .env

### ISSUE THREAD

**Issue by ElYaiko** [oven-sh/bun#9545]
### What is the problem this feature would solve?

Sometimes you want to explicitly show what variables are in your `.env` but the `Node.Process.env`  only types `NODE_ENV` and `TZ`.

### What is the feature you are proposing to solve the problem?

Bun already reads `.env`, it would be good that generates a typed extension too of what other variables are in.

### What alternatives have you considered?

Hope this can be implemented, thanks!.

**Comment by kaminskypavel** [comment]
this is not how you type environmental variables. 

if you're interested, there's a [whole section ](https://bun.sh/docs/runtime/env#typescript) in the docs how to do this

**Comment by ElYaiko** [comment]
> this is not how you type environmental variables.
> 
> if you're interested, there's a [whole section ](https://bun.sh/docs/runtime/env#typescript) in the docs how to do this

Yes but as I mentioned, it would be good that Bun generates this types extension instead of having to do it manual.
